.yamato/ruamel/jobs/abv/abv_all_project_ci.py

```diff
@@ -30,4 +30,6 @@ def get_job_definition(self, editor, projects, target_branch):
         # (Temporarily?) disable automatic runs of the ABV for PRs
         if editor.get("abv_pr"):
             job.set_trigger_on_expression(f'pull_request.target eq "{target_branch}" AND NOT pull_request.draft AND NOT pull_request.push.changes.all match ["**/*.md", "doc/**/*", "**/Documentation*/**/*", ".github/**/*", "Tools/**/*"]')
+        if editor['name'].lower() != 'custom-revision':
```

Comment:
Won't the `.lower()` part break the backports if editor["name"] resolves to 2021.1 for instance? 
We had this issue before: https://github.com/Unity-Technologies/Graphics/pull/2908/files?file-filters%5B%5D=.py#diff-53494a5c141b9b056a0ad3ee5d5254c973431e9555ac425d6d727694a57f9bf7

Reply:
Good catch, will push an update